src/promptflow/promptflow/_cli/_pf/entry.py

```diff
@@ -58,6 +61,14 @@ def entry(argv):
         if hasattr(args, "debug") and args.debug:
             for handler in logging.getLogger(LOGGER_NAME).handlers:
                 handler.setLevel(logging.DEBUG)
+
+        activity_name = f"pf.{args.action}.{args.sub_action}.cli"
+        message = f"{activity_name}.start"
+        add_telemetry_log(activity_name=activity_name,
+                          activity_type=ActivityType.PUBLICAPI,
+                          message=message,
+                          custom_dimensions={"request_id": str(uuid.uuid4())})
```

Comment:
Do we have some existing envrionment variable that control the request_id ? @D-W-

Reply:
Currently there are not